lms/tests/test_extractor.py

```diff
@@ -1,42 +1,58 @@
 import pathlib
+from tempfile import SpooledTemporaryFile
 
-import pytest
+from werkzeug.datastructures import FileStorage
 
 import lms.extractors.base as extractor
+import lms.extractors.ziparchive as zipfilearchive
 
 
-PYFILE1 = '# Upload 3141\n\nNormal exercise'
-PYFILE2 = """
-            
-                         
-# Upload 3141
+samples_dir = pathlib.Path(__file__).parent / 'samples'
 
-Normal exercise
-להלהלה
+file1 = open(f'{samples_dir}/code1.py')
```

Comment:
Move this to be a fixture. It's a bad idea to open them anyway and close them only if you `test_py` run.